SobokSobok/SobokSobok/Presentation/Medicine/AddMyMedicineView/AddMyMedicineCollectionViewCell.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  AddMyMedicineCollectionViewCell.swift
+//  SobokSobok
+//
+//  Created by 김승찬 on 2022/01/10.
+//
+
+import UIKit
+
+final class AddMyMedicineCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: - Properties
+    
+    var deleteCellClosure: (() -> ())?
+    var cellHeightClosure: (() -> ())?
+    
+    // MARK: - @IBOutlets
+    
+    @IBOutlet var rootView: UIView!
+    @IBOutlet var medicineMainView: UIView!
+    @IBOutlet var medicineTextField: UITextField!
+    @IBOutlet var deleteTextButton: UIButton!
+    @IBOutlet var deleteMedicineCellButton: UIButton!
+    @IBOutlet var writingCountLabel: UILabel!
+    @IBOutlet var medicineCellHeight: NSLayoutConstraint!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        setUI()
+        setDeleteButton()
+        assignDelegate()
+    }
+    
+    // MARK: - Functions
+    
+    private func setUI() {
+        medicineMainView.makeRoundedWithBorder(radius: 10, color: Color.gray300!.cgColor)
+    }
+    
+    private func assignDelegate() {
+        medicineTextField.delegate = self
+    }
+    
+    private func setDeleteButton() {
+        medicineTextField.addTarget(self, action: #selector(AddMyMedicineCollectionViewCell.medicineTextFieldDidChange(_:)), for: UIControl.Event.allEditingEvents)
+    }
+    
+    // Color, Bool을 매개변수로 가지는 함수
+    private func checkMedicineTextField(color: CGColor, bool: Bool) {
+        medicineMainView.layer.borderColor = color
+        deleteTextButton.isHidden = bool
+        deleteMedicineCellButton.isHidden = true
+        writingCountLabel.isHidden = false
+    }
+    
+    private func changeUI() {
```

Comment:
```suggestion
    private func updateUI() {
```
change보다 update로 통일하면 좋을 것 같아요~!